fix: corrections in pilot-corner (#771)
flybywiresim/docs: docs/pilots-corner/beginner-guide/powering-down.md

    ![Parking Brake](../assets/beginner-guide/powering-down/parking-brk.png "Parking Brake"){loading=lazy}

> - `NOSE TAXI` and `RWY TURN OFF` lights are usually turned off even before we turn into the gate to not blind the ground personnel. This, of course, is only done if enough lighting is available to safely park into the gate. Turn them off if they are used to assisting in parking.

Reviewer: ```suggestion
- `NOSE TAXI` and `RWY TURN OFF` lights are usually turned off even before we turn into the gate to not blind the ground personnel. This, of course, is only done if enough lighting is available to safely park into the gate. Turn them off if they are not going to be used to assist in parking.
```

Author: ```suggestion
- `NOSE TAXI` and `RWY TURN OFF` lights are usually turned off even before we turn into the gate to not blind the ground personnel. This, of course, is only done if enough lighting is available to safely park into the gate. Turn them off if they were used to while parking.
```

Considering the page this is probably a better version since the plane will already at the gate or parking stand.
